interp/constrexpr_ops.mli

```diff
@@ -124,3 +124,10 @@ val error_invalid_pattern_notation : ?loc:Loc.t -> unit -> 'a
 
 (** Placeholder for global option, should be moved to a parameter *)
 val asymmetric_patterns : bool ref
+
+(** Local universe and constraint declarations. *)
+val interp_univ_decl : Environ.env -> universe_decl_expr ->
+                       Evd.evar_map * universe_decl
+
+val interp_univ_decl_opt : Environ.env -> universe_decl_expr option ->
```

Comment:
The thing is that the type is only used in the AST tho, so I dunno, we could indeed put `universe_decl_expr` in the core pretyping AST `Glob_term` but it would be then unused there.